opensearch/main.tf

```
@@ -89,16 +89,18 @@ resource "aws_security_group" "opensearch-security-group" {
 }
 
 resource "random_password" "password" {
-  length      = 32
-  upper       = true
-  special     = true
-  lower       = true
-  numeric     = true
-  min_upper   = 1
-  min_special = 1
-  min_lower   = 1
-  min_numeric = 1
+  length           = 32
+  upper            = true
+  special          = true
+  lower            = true
+  numeric          = true
+  min_upper        = 1
+  min_special      = 1
+  min_lower        = 1
+  min_numeric      = 1
+  override_special = coalesce(var.config.password_special_characters, "-_!.~$&'()*+,;=")
```

Override special what?
```suggestion
  override_special_characters_allowed_in_password = coalesce(var.config.password_special_characters, "-_!.~$&'()*+,;=")
```

---

characters. It's not my definition, its part of the random_password resource

---

Fair 😄